Testreglar/4.1.2/Nett/nett-4.1.2b.json

```diff
@@ -0,0 +1,314 @@
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.8"
+				}
+			}
+		},
+		{
+			"stegnr": "3.8",
+			"spm": "Hvor mange tilstander har knappen? ",
+			"ht": "<p>Registrer antall tilstander.</p>\n<p>Eksempel: En spill av og stopp knapp kan ha to tilstander, spill av eller stopp.</p>",
+			"type": "tekst",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.9"
+				}
+			},
+			"label": "Antall tilstander:",
+			"filter": "tal"
+		},
+		{
+			"stegnr": "3.9",
+			"spm": "Hvor mange tilstander er ikke angitt programmatisk?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Inspiser knappen i Chrome</li>\n<li>Bruk Accessibility Checker verktøy i inspiser</li>\n<li>Under ARIA-attributter eller Computed Properties, sjekk om det finnes:<br>\n<ul>\n<li>For av/på knapp (toggle button) og spill av og stopp knapp:<br>\n<ul>\n<li>hvis knappen er på: aria-pressed: true eller aria-checked: true</li>\n<li>hvis knappen er av: aria-pressed: false eller aria-checked: false</li>\n<li>hvis knappen har en mellomliggende tilstand mellom tilstander av og på: aria-pressed: mixed.</li>\n</ul>\n</li>\n<li>en knapp som utvider eller sammenslutter innhold<br>\n<ul>\n<li>hvis innholdet er sammensluttet: aria-expanded: false</li>\n<li>hvis innholdet er utvidet: aria-expanded: true</li>\n<li>har attributtet: aria-haspopup: true </li>\n<li>har attributtet:  aria-controls: \"IDREF\"\n<ul>\n<li>IDREF refererer til verdien i attributtet \"id\" til elementet med menyen som styres av meny-knappen.</li>\n</ul>\n</li>\n</ul>\n</li>\n</ul>\n</li>\n<li>Registrer antall tilstander som ikke er angitt programmatisk.</li>\n</ul>",
+			"type": "tekst",
+			"ruting": {
+				"alle": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"type": "lik",
+							"sjekk": "3.9",
+							"verdi": "0",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Ja",
+								"utfall": "Knappen har role button, et tilgjengelig navn som beskriver formålet med knappen og tilstanden er angitt programmatisk."
+							}
+						},
+						"2": {
+							"type": "ulik",
+							"sjekk": "3.9",
+							"verdi": "0",
+							"handling": {
+								"type": "gaaTil",
+								"steg": "3.10"
+							}
+						}
+					}
+				}
+			},
+			"kilde": [
+				"ARIA5",
+				"G10"
+			],
+			"label": "Antall tilstander:",
+			"filter": "tal"
+		},
+		{
+			"stegnr": "3.10",
+			"spm": "Hvilke tilstander er ikke angitt programmatisk?",
+			"ht": "<p>Beskriv tilstander som ikke er angitt programmatisk, slik at det er mulig å identifisere dem i ettertid.</p>",
+			"type": "tekst",
+			"ruting": {
+				"alle": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Knappen har role button og et tilgjengelig navn som beskriver formålet med knappen, men tilstanden er ikke angitt programmatisk."
+				}
+			},
+			"label": "Tilstander:",
+			"multilinje": true
+		},
+		{
+			"stegnr": "3.11",
+			"spm": "Har testsiden knapper som verktøyet ikke avdekker?",
```

Comment:
Hvis vi allerede er usikre på om det er knapper som ikke blir avdekket så bør vi ikke bruke verktøyet.

Reply:
@janBeniaminKwiek Vi er ikke usikre på det, Vi vet at QW bare finner knapper som er kodet med rolle knapp. Det er også presisert i hjelpeteksten.

Reply:
Steg for bruk av verktøyet har blitt fjernet.